tests/web_api/test_variables.py

```diff
@@ -113,13 +113,13 @@ def test_variable_with_enum():
     response = subject.get('/variable/housing_occupancy_status')
     variable = json.loads(response.data)
     assert_equal(variable['valueType'], 'String')
-    assert_equal(variable['defaultValue'], 'Tenant')
+    assert_equal(variable['defaultValue'], 'tenant')
     assert_in('possibleValues', variable.keys())
     assert_equal(variable['possibleValues'], [
```

Comment:
I think the order matters because the possible values will be displayed in the legislation explorer through the API . I think it will be easier for users to use it if the items of the enums are always displayed in the same order

Reply:
Requesting the description knowing the id using an object:

```js
variable.possibleValues.free_lodger
```

Using an array:

```js
variable.possibleValues.find(function(item) {
  return item.name == 'free_lodger'
})
```

For the user, the array is much more annoying to request. Using an array makes sense only if the order actually _means_ something. It doesn't in our case. 

If you want the legislation explorer to always show the item in the same order:
- Either you're pessimistic, and you implement yourself some kind of alphabetical sorting.
- Either you're optimisitc, and you consider that even though objects don't have an order, browsers and flask tend to serialise and deserialise them in a deterministic way. Which means than _most likely_, the order in which the items will show will stay consistent.